### ISSUE TITLE
Add-on version stats for Android all show up as Unknown version

### ISSUE THREAD

**Issue by ghostwords** [mozilla/addons-server#15372]
### Describe the problem and steps to reproduce it:
There is now an "Unknown" version in the Add-on Version tab (under `.../statistics/usage/versions/`) of your add-on's stats dashboard.

Furthermore, the number of Android users (under `.../statistics/usage/os/`) seems to precisely match the number of Unknown version users.

### What happened?

Some users are being reported as using Unknown version of your add-on.

### What did you expect to happen?

There is no Unknown version.

### Anything else we should know?

It seems that all Android users are reported as Unknown version users.

**Comment by ghostwords** [comment]
This is happening in the Privacy Badger dashboard, by the way.

**Comment by jvillalobos** [comment]
Yeah, this is known issue. It needs to be fixed in the Fenix telemetry ping, which is being tracked here: https://github.com/mozilla-mobile/fenix/issues/13322. Keeping this open so we track it on our end as well.

**Comment by KevinMind** [comment]
Old Jira Ticket: https://mozilla-hub.atlassian.net/browse/ADDSRV-45
